### PR TITLE
#1161 - Updated status badges for build result and code coverage

### PR DESCRIPTION
**Pull request by pdamodaran**
This PR addresses some of the remaining TODOs in  #1161 

I have made the following changes:

1.  Replaced Travis CI status badge with GitHub Actions status badge
2. Replaced Coveralls status badge with Codecov status badge
3. Changed GitHub Actions workflow name

Still to do:

- Update any references to Coveralls and Travis
- Fix issue with slack notification if it is still a problem after this PR

Questions for the @DistrictDataLabs/team-oz-maintainers:

- Should the status badge for AppVeyor be removed?
- Are there any other TODOs that come in mind other than the ones mentioned above?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Yellowbrick
 
-[![Build Status](https://travis-ci.com/DistrictDataLabs/yellowbrick.svg?branch=develop)](https://travis-ci.com/DistrictDataLabs/yellowbrick)
-[![Build status](https://ci.appveyor.com/api/projects/status/11abg00ollbdf4oy?svg=true)](https://ci.appveyor.com/project/districtdatalabs/yellowbrick)
-[![Coverage Status](https://coveralls.io/repos/github/DistrictDataLabs/yellowbrick/badge.svg?branch=master)](https://coveralls.io/github/DistrictDataLabs/yellowbrick?branch=master)
+[![Build Status](https://github.com/DistrictDataLabs/yellowbrick/actions/workflows/ci.yml/badge.svg?branch=develop)](https://github.com/DistrictDataLabs/yellowbrick/actions/workflows/ci.yml)
+[![Build Status](https://ci.appveyor.com/api/projects/status/11abg00ollbdf4oy?svg=true)](https://ci.appveyor.com/project/districtdatalabs/yellowbrick)
```
Comment (bbengfort) [inline README.md:4]: We could probably remove the AppVeyor badge since we don't use AppVeyor CI anymore either.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Python package
+name: Yellowbrick CI Workflow
```
Comment (bbengfort) [inline .github/workflows/ci.yml:4]: I like this - thank you for adding it!

**Review by bbengfort — APPROVED** [review]
@pdamodaran it looks like the CI checks are working now - that's great news! Thank you and @ndanielsen for taking this on and adding GitHub actions for our CI workflow. Adding the badge to the README is an excellent detail, and I appreciate the challenge of dealing with all of the commits and minor issues with CI and dependency versions!

File: README.md (hunk 1 above)
Comment (pdamodaran) [reply README.md]: I have removed this badge

**Comment by bbengfort** [comment]
@pdamodaran thank you so much! I'm not sure if there is anything else to work on here, but I'm going to merge it in an effort to get the tests passing in #1172

**Pull request merged**